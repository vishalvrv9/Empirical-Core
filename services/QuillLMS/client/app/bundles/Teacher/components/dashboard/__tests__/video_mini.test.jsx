import React from 'react';
import { shallow } from 'enzyme';

import VideoMini from '../video_mini';

describe('VideoMini component', () => {

  it('should render', () => {
    const wrapper = shallow(
      <VideoMini videoCode='https://www.youtube.com/embed/dQw4w9WgXcQ' />
    );
    expect(wrapper).toMatchSnapshot();
  });

});
