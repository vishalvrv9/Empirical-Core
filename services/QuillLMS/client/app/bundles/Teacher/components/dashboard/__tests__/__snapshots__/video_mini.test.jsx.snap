// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoMini component should render 1`] = `
<div
  className="mini_container  col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="video-container"
    >
      <h4>
        Building Better Writers
      </h4>
      <iframe
        allowFullScreen={true}
        frameBorder="0"
        height="168"
        src="https://www.youtube.com/embed/dQw4w9WgXcQ?&theme=light&autohide=2&showinfo=0"
        width="298"
      />
      <p>
        An overview of how Quill works.
      </p>
    </div>
  </div>
</div>
`;
